From b6c170cc5dd1bc677176ee732cdb0ddae57e5cf0 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Fri, 25 Oct 2019 15:10:26 +0000
Subject: [PATCH] Add missing header bootlegged by Boost < 1.72

src/parser.y:76:6: error: no template named 'stack' in namespace 'std'
std::stack<LocalScope *> scope_stack;
~~~~~^
---
 src/parser.y | 1 +
 1 file changed, 1 insertion(+)

diff --git src/parser.y src/parser.y
index 7f4fd56ca7..4c77c989ea 100644
--- src/parser.y
+++ src/parser.y
@@ -46,6 +46,7 @@
 #include "printutils.h"
 #include "memory.h"
 #include <sstream>
+#include <stack>
 #include <boost/filesystem.hpp>
 #include "boost-utils.h"
 #include "feature.h"
